benchcab/utils/__init__.py

```diff
@@ -0,0 +1,49 @@
+"""Top-level utilities."""
+import pkgutil
+import json
+import yaml
+import os
+from importlib import resources
+from pathlib import Path
+
+
+# List of one-argument decoding functions.
+PACKAGE_DATA_DECODERS = dict(
+    json=json.loads,
+    yml=yaml.safe_load
+)
+
+
+def get_installed_root() -> Path:
+    """Get the installed root of the benchcab installation.
+
+    Returns
+    -------
+    Path
+        Path to the installed root.
+    """
+    return Path(resources.files('benchcab'))
```

Comment:
I think I do something similar here:
https://github.com/CABLE-LSM/benchcab/blob/6ed4cd755721893fe737d4a66026af00390b33fe/benchcab/benchcab.py#L323

We can replace the argument to `benchcab_exe_path` to call your utility function?

In the above I preferred to use `sys.argv[0]` since we might want to invoke a path to a particular installation of benchcab (for example the conda installation), then benchcab would know which installation it is.

Reply:
Unless I've misunderstood your function (in that the installed location is not the path to the benchcab executable)?